exotic/exotic.py

```diff
@@ -1686,6 +1687,13 @@ def main():
         else:
             pDict = userpDict
             CandidatePlanetBool = False
+        # Seed random number generator (for run to run consistency)
+        if exotic_infoDict['random_seed']:
+            log_info(f"Setting random number seed to {exotic_infoDict['random_seed']}")
+        else:
+            exotic_infoDict['random_seed'] = int.from_bytes(hashlib.sha256(f"{pDict['pName']}:{pDict['midT']}".encode()).digest()[0:4], byteorder='little')
```

Comment:
Definitely no problem with displaying the fixed seed - I was kind of on the fence on whether to add the 'noise' or not.  I did want to display the non-fixed seed because it does make the value used part of the logs - allowing for the value to be opted to be fixed later, for future runs, for example.

The point IS ultimately to enable run to run consistency - which is generally viewed as a positive thing when it comes to research output artifacts: without it, sharing data, code and methodology is insufficient to assure reproducibility, both by ourselves and by others.   For any given run, a deterministic seed is used - whether provided by us systemically, or via the seed initialization in the numpy runtime (which is neither deterministic nor repeatable, as current time and other entropy sources are generally used for such things) - the result is only the randomness of the selection of sequence used, not the quality of the randomness within that sequence: if the quality of the RNG sequence algorithm is poor, using a fixed or uncontrolled seed will not fix it.